[Question] Why "@preact/playwright-ct" types are not available?

I have a project with preact and I installed the package `preact/playwright-ct`, however, Im seeing this error when importing config on my `config.ts` file :
        "resource": "/playwright-ct.config.ts",
	"owner": "typescript",
	"code": "2307",
	"message": "Cannot find module '@preact/playwright-ct' or its corresponding type declarations."


Does anyone know how to fix it? is there a missing dependency , maybe?
@developit

--- a/package.json
+++ b/package.json
@@ -10,6 +10,7 @@
   ],
   "author": "Marvin Hagemeister <hello@marvinh.dev>",
   "license": "MIT",
+  "types": "./dist/index.d.ts",
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",

